scripts/cypress/cypress.config.ts

```diff
@@ -10,6 +10,19 @@ const applyV8WebpackConfig: (config: Configuration) => Configuration = require('
 
 const isLocalRun = !process.env.DEPLOYURL;
 
+function isV8() {
+  const packageJsonPath = path.join(process.cwd(), 'package.json');
```

Comment:
this assumes that the cwd is in the root of a package, perhaps use `findUp` ?

Reply:
curios why  ? cypress is always invoked from root of a package

Reply:
I see, well that's definitely not supported behaviour as it adds magic to our DX. you should execute task always from root. With nx in place this will be kinda enforced as well.